src/views/blocks/event-links.php

```diff
@@ -3,17 +3,18 @@
  * Block: Event Links
  *
  * Override this template in your own theme by creating a file at:
- * [your-theme]/tribe/events/blocks/event-links` *
+ * [your-theme]/tribe/events/blocks/event-links.php
+ *
  * See more documentation about our Blocks Editor templating system.
  *
- * @link http://evnt.is/1aiy
+ * @link https://evnt.is/1aiy
  *
  * @version 4.7
  *
  * @var Tribe__Events__Editor__Template $this
  */
 
-// don't show on password protected posts
+// Don't show on password protected posts.
```

Comment:
Is this in the wrong place? Should be above line 20

Reply:
Totally valid. Moved.